apps/projects/app/components/Content/Issues.js

```diff
@@ -228,10 +228,19 @@ class Issues extends React.PureComponent {
     </StyledIssues>
   )
 
+  getExpLevels = () => {
+    let expLevels = []
+    let a = this.props.bountySettings.expLevels.split('\t')
+    for (let i = 0; i < a.length; i += 2)
+      expLevels.push({ mul: a[i] / 100, name: a[i + 1] })
```

Comment:
I personally rather to use better descriptive variable names, it helps others to follow code, like `multiplier` instead of `mul` but just an opinion, feel free to keep that way

Reply:
Acknowledged. This particular piece of code will disappear soon though - I'm leaving it unchanged for now.